src/component/Cursor.tsx

```diff
@@ -98,16 +101,17 @@ export function CursorInternal(props: CursorConnectedProps) {
  */
 export function Cursor(props: CursorProps) {
   const tooltipAxisBandSize = useTooltipAxisBandSize();
-  const { coordinate, payload, index } = useTooltipContext();
+  // TODO: select from redux
+  const { index } = useTooltipContext();
   const offset = useOffset();
   const layout = useChartLayout();
   const chartName = useChartName();
   return (
     <CursorInternal
       {...props}
-      coordinate={coordinate}
+      coordinate={props.coordinate}
       index={index}
-      payload={payload}
+      payload={props.payload}
```

Comment:
@PavelVanecek was there a reason these came from from context when they can just be passed in?

Reply:
This was a refactor in progress. Cursor was first and Tooltip second. This change should be safe.